src/sass/patterns/_lists.scss

```diff
@@ -11,7 +11,22 @@ li {
 [data-list='inline'] {
   align-items: center;
   display: flex;
-  gap: var(--gutter);
+  gap: var(--shim);
+}
+
+details {
+  --link: var(--fgcolor);
+  --link-focus: var(--fgcolor);
+
+  background-color: var(--status-result-bg, var(--bgcolor));
+  border-radius: var(--border-radius);
+  color: var(--status-result-fg, var(--fgcolor));
+}
+
+summary {
+  cursor: pointer;
+  font-weight: bold;
+  list-style-position: outside;
```

Comment:
@dvdherron On the dl below, I wonder if we will want to add something like dl:last-child and give it a zero margin-block-end. Otherwise the spacing gets a little large between sections since the gap is already adding space to the grid rows.